skt/executable.py

```diff
@@ -509,21 +510,21 @@ def setup_parser():
         type=str,
         action='append',
         help="Path to a local patch to apply "
-             + "(use multiple times for multiple patches)"
+        + "(use multiple times for multiple patches)"
```

Comment:
While this line (and those below) fix the pylint warning, the changes make the code less readable since it's not easy to see where the help actually ends. If you can solve the warning in a way it stays clear that would be more than awesome, if not, I'd prefer for it to stay as it was, even if it produces a warning.

Reply:
I am not sure what you mean `the changes make the code less readable since it's not easy to see where the help actually ends` ? 
Furthermore, I am not sure there are some different format for `help`. For example: https://github.com/RH-FMK/skt/blob/master/skt/executable.py#L531

Reply:
I do not know the name for `mb`. Could you give the name for it? https://github.com/RH-FMK/skt/blob/master/skt/executable.py#L152

Reply:
@danghai please check the git history of why the formatting was changed, and the internet for "how to split long string in Python" which should show you both of these (and maybe some others too) options with description.

> I am not sure what you mean the changes make the code less readable since it's not easy to see where the help actually ends ?

Python is based on code indentation and readability, as I'm sure you noticed. If the continuation line is indented same as the line above, it doesn't look like continuation line but separate statement on the first look.

Reply:
@veruu I see the reason why in commit `Fix placing of binary operators`. So I will keep the same indentation in spite of the warning in `pylint`.